everyBot/cogs/math/math.py

```diff
@@ -0,0 +1,237 @@
+import discord
+from discord.ext import commands
+from discord.ext.commands import has_permissions
+from pymongo.errors import ServerSelectionTimeoutError
+
+from .. import database

```

Comment:
I know in most of the files this is how the database is imported but i'd like to change it to:
`from everyBot.cogs import database`, it's a better way of doing it.

Reply:
Done 👍